package.json

```diff
@@ -100,20 +100,16 @@
     },
     "publish": {
       "provider": "github",
-      "owner": "TEZSURE GUI V2.0",
-      "repo": "TEZSURE GUI V2.0",
+      "owner": "TEZSURE GUI",
```

Comment:
Owner is 'Tezsure Inc' 
please make this change wherever required. 
also change 'repo' to appropriate value.